Netkan/Sources/Curse/CurseMod.cs

```diff
@@ -10,22 +10,24 @@ internal class CurseMod
     {
         [JsonProperty] public string license;
         [JsonProperty] public string title;
-        //[JsonProperty] public string short_description;
```

Comment:
We can pull the `description` from curse if you uncomment line 81 in `NetKAN/Transformers/CurseTransformer.cs`.

Reply:
Also in `CurseTransformer.cs` change `curseMod.authors` to `curseMod.members` on line 88.

Reply:
Cool, good idea. I assume we need to remove the `short_` part to match the API. Will do.

`curseMod.members` is a list of objects with more data than just the list of strings needed by `author`, so we can't use them interchangeably. That's why `curseMod.authors` is now replaced with a property that analyzes `members`.

Reply:
From a "I've not looked at this code or the new API" perspective, I'm generally in favour of moving to new APIs sooner rather than later unless there is a compelling reason not to.